shell/platform/linux/fl_dart_project_test.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_dart_project.h"
+#include "gtest/gtest.h"
+
+#include <gmodule.h>
+
+TEST(FlDartProjectTest, GetPath) {
+  g_autoptr(FlDartProject) project =
+      fl_dart_project_new("/projects/my_dart_project");
+  g_assert_cmpstr(fl_dart_project_get_path(project), ==,
```

Comment:
Does mixing GLib asserts and gtest work correctly? It seems like it would be clearer to use gtest's expectation and assertion macros.

Reply:
They do work, but muscle memory had me type the GLib ones. I've updated them to the gtest equivalents.